app/routes/git_tokens.py

```
@@ -86,9 +89,11 @@ async def handle_gitlab(
         return APIResponse.success(
             message=constants.TOKEN_SAVED_SUCCESSFULLY, data={"id": str(git_label.id)}
         )
-    except Exception:
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)